galgebra/_cached_property.py

```diff
@@ -0,0 +1,29 @@
+
+class cached_property:
```

Comment:
Can this file be placed somewhere else? Don't like `_*.py` to appear at the root, they would confuse first comers if there'll be more...

P.S. Is it ported or backported from somewhere? Then `_backports` might fit? (B.T.W. I didn't like this restrictive name last time too, these small utility functions or classes are ported here to keep the dependency minimal, they deserve a dedicated place).

Reply:
So, the purpose of `_*.py` is to indicate that the module is private, and for internal use only. If this were called `galgebra.cached_property`, then that would imply to users that they can use it themselves, and would mean that we would have to support it forever.

So I'm quite attached to using an `_` at the start of the filename, but I'd also be happy to pick a more generic filename to let us pile more things in here.

Reply:
`_core/cached_property.py`?

Reply:
`galgebra/_utils/cached_property.py` 😄

Reply:
Done